test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.Netcore.csproj

```diff
@@ -11,6 +11,10 @@
     <OutputPath>..\..\..\bin\AnyCPU\$(Configuration)\Test\.NETPortable</OutputPath>
     <DefineConstants>$(DefineConstants);PORTABLELIB;</DefineConstants>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Tracking\DataServiceContextNoTrackingModeStreamsTests.cs" />

```

Comment:
Any reason for removing this from the project?

Reply:
I removed this class and forgot to edit this. The reason for removal was this class tested the non-async api which wouldn't work in .net core